himan-scripts/CB-TCU-cloud.lua

```diff
@@ -64,10 +74,11 @@ for i=1, #EL500 do
   res[i] = Missing
 
   --TCU
-  if ((Tbase[i]-Ttop[i]>TCUlimit) and (NL[i]>0) and (CIN500[i]>CINlimTCU)) then
-    res[i] = FlightLevel_(EL500[i]*100)
+  if (EL500[i] - LCL500[i] > TCUlimit) then
```

Comment:
In smarttool the if condition is:

if ((EL500-LCL500>TCulimit) and (cl_meps>0) and (CIN500>CINlimTCu))

The two latter boolean conditions are missing